### ISSUE TITLE
Libnd4j: need sparse softmax cross entropy loss with logits

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#6255]
Required for TF import.

It's the same as softmax_cross_entropy_loss_with_logits op, except for the labels format:
in softmax_cross_entropy_loss_with_logits, labels are one-hot, and predictions/labels have same rank
in sparse_softmax_cross_entropy_loss_with_logits, labels are integer indices, and labels have rank of 1 less than the one-hot.

For example, labels shape [10], predictions shape [10,5] (i.e., [minibatch, nClasses], with labels values being in range 0 (inclusive) to nClasses=5 (exclusive).

**Comment by raver119** [comment]
There's no real sense of posting 1 issue for every missing op, since we have list of all TF ops available. Maybe you could compile new ops-we-want list with priorities like we had back in days?

**Comment by raver119** [comment]
Or, alternatively, put some priority tag for this op, so we can properly schedule cycles for implementation?

**Comment by shyrma** [comment]
this loss has been implemented and corresponding pr is already merged into master

**Comment by AlexDBlack** [comment]
Thanks! I've added Java class, TF import mapping and tests on my branch, all confirmed passing.
